plugins/org.jboss.tools.openshift.ui/META-INF/MANIFEST.MF

```diff
@@ -28,6 +28,10 @@ Export-Package: org.jboss.tools.openshift.internal.ui;x-friends:="org.jboss.tool
  org.jboss.tools.openshift.internal.ui.command;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.deployment;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.explorer;x-friends:="org.jboss.tools.openshift.test",
+ org.jboss.tools.openshift.internal.ui.job,
```

Comment:
please add x-friends since I would think that jobs are internal packages (as the package name suggests)